src/components/Quotes.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+
+const Quotes = () => {
+  const [data, setData] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+
+    fetch('https://api.api-ninjas.com/v1/quotes?category=funny',
+      {
+        // mode: 'no-cors',
+        headers: {
+          'X-Api-Key': 'hTAKXNFRMzvu2GX5y3pZ0Jo8X9WRprAcTGQV9DPv',
+          'Content-Type': 'application/json',
+        },
+      })
+      .then((res) => res.json())
+      .then((data) => {
+        setData(data);
+      })
+      .catch((error) => setError(error.message));
+    setLoading(false);
+  }, [setData, setLoading]);
+
+  if (loading) return <h2>Loading... please wait</h2>;
+  if (error) {
```

Comment:
- [ ] Great job creating a There is a `loading` state 👏🏻 However it does not show up while fetching data from the API, Please make this working as expected 👍🏻